django/forms/fields.py

```diff
@@ -750,7 +750,8 @@ class URLField(CharField):
     }
     default_validators = [validators.URLValidator()]
 
-    def __init__(self, **kwargs):
+    def __init__(self, assume_scheme="https", **kwargs):
```

Comment:
This is a breaking change (unfortunately 😅 )
I think it should remain `http` and raise a `RemovedInDjango60Warning`, and then we can change it to `https`.

Reply:
An alternative could be that we look for `assume_scheme` in kwargs perhaps? Though I guess that will still be breaking change, since you change behaviour.

Reply:
It's better to use a sentinel value to detect when unspecified. Also the arg should be [kwarg only](https://lukeplant.me.uk/blog/posts/keyword-only-arguments-in-python/)

```suggestion
    def __init__(self, *, assume_scheme=UNSPECIFIED, **kwargs):
        if assume_scheme is UNSPECIFIED:
            warnings.warn(...)
            assume_scheme = "https"
```

Reply:
Thank you for the review. I've updated the PR.